### ISSUE TITLE
DropDownListComponent.value = string does not update component

### ISSUE THREAD

**Issue by cluetjen** [telerik/kendo-angular#1365]
## I'm submitting a...

* Bug report 

## Current behavior

`myDropDownList.value = "someid";`  does not update the visible value of control.

See https://plnkr.co/edit/tSgcSX2iy2IxR0Rs4klo?p=preview (app.component.ts > ngAfterViewInit) for details, but in short:
* kendo-dropdownlist
* data: { text: string, value: number }[];
* textField=text, valueField=value, valuePrimitive=true

```
    ngAfterViewInit() {
        this.dropDown.value = 2;
    }
```

## Expected behavior

Component should show value with "someid" as "current value".

## Minimal reproduction of the problem with instructions
https://plnkr.co/edit/tSgcSX2iy2IxR0Rs4klo?p=preview

* Open plunker in browser
* => DropDownList should show "Medium" but it's empty.

## What is the motivation or use case for changing the behavior?

I've to create / respond to an event, that's why I don't use data binding but use valueChange event to subscribe to changes and tried to use cmp.value = "idfromevent" to update the ui.

## Environment
progress/kendo-angular-dropdowns@2.1.0

Browser:
Tested with Chrome,64.0.3282.186

System:
- TypeScript version: 2.6.2
- Node version: 6
- Platform:  Win 10

**Comment by ipeshev** [comment]
Fix available in [latest dev version](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/).

**Comment by stnikolova** [comment]
Released in **v4.4.0.**
